[enhancement] Update README with installation instructions

exatonix opened the pull request:
Added installation instructions and system requirements for Debian 13 with Latest Python3 (3.13 Installed)

exatonix committed:
Update README with installation instructions

Added installation instructions and system requirements for Debian 13 with Latest Python3 (3.13 Installed)

p0dalirius commented on README.md:
```diff
@@ -45,3 +45,19 @@ optional arguments:
+git clone https://github.com/p0dalirius/WifiListProbeRequests.git
+cd WifiListProbeRequests
+
+## Fixes  (<a href="https://https://gitub.com/exatonix"</a>)
```
Can you remove this? `(<a href="https://https://gitub.com/exatonix"</a>)`

exatonix replied:
ı did yesterday night

p0dalirius commented on README.md:
````diff
@@ -45,3 +45,19 @@ optional arguments:
+```bash
+git clone https://github.com/p0dalirius/WifiListProbeRequests.git
+cd WifiListProbeRequests
+
````
There is the end of the code bloc missing there \`\`\`

exatonix committed:
Fix typos and formatting in README.md

Corrected typos and formatting in the README.

Lost Installation Method Fixed.
Some Dependency and Packages Explained and Added

exatonix commented:
ı  first time make a big project sorry for late answer. ı saw but answer
too late . ı fixed what you said. look again

20 Kas 2025 Per 13:32 tarihinde Rémi GASCOU (Podalirius) <
***@***.***> şunu yazdı:

> ***@***.**** requested changes on this pull request.
> ------------------------------
>
> In README.md
> <https://github.com/p0dalirius/WifiListProbeRequests/pull/2#discussion_r2545339126>
> :
>
> > @@ -45,3 +45,19 @@ optional arguments:
>  ## Contributing
>
>  Pull requests are welcome. Feel free to open an issue if you want to add other features.
> +
> +## Installation
> +
> +Clone the repository and install dependencies:
> +
> +```bash
> +git clone https://github.com/p0dalirius/WifiListProbeRequests.git
> +cd WifiListProbeRequests
> +
> +## Fixes  (<a href="https://https://gitub.com/exatonix"</a>)
>
> Can you remove this? (<a href="https://https://gitub.com/exatonix"</a>)
> ------------------------------
>
> In README.md
> <https://github.com/p0dalirius/WifiListProbeRequests/pull/2#discussion_r2545340820>
> :
>
> > @@ -45,3 +45,19 @@ optional arguments:
>  ## Contributing
>
>  Pull requests are welcome. Feel free to open an issue if you want to add other features.
> +
> +## Installation
> +
> +Clone the repository and install dependencies:
> +
> +```bash
> +git clone https://github.com/p0dalirius/WifiListProbeRequests.git
> +cd WifiListProbeRequests
> +
>
> There is the end of the code bloc missing there ```
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/p0dalirius/WifiListProbeRequests/pull/2#pullrequestreview-3486944795>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACIIR3Z24FSGWUKZ6DJVTSD35WKCBAVCNFSM6AAAAACMI5IFUSVHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZTIOBWHE2DINZZGU>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
